### ISSUE TITLE
Gmail labels with accents are indexed twice

### ISSUE THREAD

**Issue by neurolit** [nylas/sync-engine#221]
Hi,

In N1, each Gmail label with accented letters (ë for example) is displayed twice. I am using a local sync-engine, so I used the API in order to find if it was caused by N1 or sync-engine:

```shell
curl --basic -u 'xxx:xxx' http://localhost:5555/labels
```
The response shows that sync-engine stores two different labels instead of one ("1. Gaëlle"):
```json
[
    {
        "account_id": "33td6cqouul3n3z9awmwq6itn",
        "display_name": "1. Ga\u00eblle",
        "id": "3k42qe06500takwjl75602r4s",
        "name": null,
        "object": "label"
    },
    {
        "account_id": "33td6cqouul3n3z9awmwq6itn",
        "display_name": "1. Ga&AOs-lle",
        "id": "eutt1arpbm3mlf7vfq4jhwg0r",
        "name": null,
        "object": "label"
    }
]
```

The second one is the good one (stores threads), according the API:
```shell
$ curl --basic -u 'xxx:xxx' http://localhost:555/threads?in=3k42qe06500takwjl75602r4s
[]
$ curl --basic -u 'xxx:xxx' http://localhost:555/threads?in=eutt1arpbm3mlf7vfq4jhwg0r
[
    {...}
]
```

Funny thing: the first one is displayed correctly in N1 ("Gaëlle), whereas the second one is wrong ("Ga&AOs-lle").

**Comment by neurolit** [comment]
Could be related to #212

**Comment by neurolit** [comment]
Solved by fdeee59 and verified on my self-hosted sync-engine.
